### PR TITLE
Fixes #30051 - Set rubocop version to same as for Foreman

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

**Comment by stejskalleos** [comment]
[test foreman_remote_execution]

File: foreman_remote_execution.gemspec [hunk 1]
```diff
@@ -29,6 +29,6 @@ Gem::Specification.new do |s|
   s.add_dependency 'foreman-tasks', '>= 0.15.1'
 
   s.add_development_dependency 'factory_bot_rails', '~> 4.8.0'
-  s.add_development_dependency 'rubocop'
+  s.add_development_dependency 'rubocop', '~> 0.85'
```
Comment (adamruzicka) [inline foreman_remote_execution.gemspec:32]: This means `0.85 <= $version < 1.0`. I'd rather stay on the safe side and go with `~> 0.85.0`

**Comment by mmoll** [comment]
With the current master tests again also running on Jenkins, this won't work with the current plugin test setup.

**Comment by stejskalleos** [comment]
[test foreman_remote_execution]

**Comment by adamruzicka** [comment]
@mmoll that means we should configure rubocop for rex the same way as in core, correct?

**Comment by mmoll** [comment]
@adamruzicka Yes, also the versions need to be the same. :disappointed:

**Comment by stejskalleos** [comment]
Rubocop updated to Foreman's version.

**Comment by adamruzicka** [comment]
Thank you @stejskalleos !

**Pull request merged**